docs/faq.md

```diff
@@ -0,0 +1,204 @@
+* The lower the version, the more platforms you can run on.
+
+So generally speaking, you should target the lowest version you get away with.
+The [version document][netstandard-versions] will help inform your decision.
+
+## How does .NET Standard versioning work?
+
+Think of the .NET Standard versions as concentric circles: higher versions
+incorporate all APIs from previous versions.
+
+## Is the API set of a .NET Standard version fixed?
+
+Yes. A specific version of .NET Standard remains frozen once shipped. New APIs
+will first become availabe in specific .NET platforms, such as .NET Core. If we
+believe the new APIs should be made available everywhere, we'll create a new
+.NET Standard version.
+
+## Is .NET Standard specific to C#?
+
+There is nothing language specific about .NET Standard. From a language view
+point, the only tie-in to .NET Standard are the language-specific runtime APIs
+(e.g. `Mirosoft.CSharp`, `Microsoft.VisualBasic`, `FSharp.Core` etc) and the
+project templates that allow you to target .NET Standard.
+
+We don't plan to add any language-specific runtime APIs to .NET Standard. The
+expectation is that they sit on top of .NET Standard and are referenced as
+needed, for example, from the project template.
+
+## Is `AppDomain` part of .NET Standard?
+
+The `AppDomain` type is part of .NET Standard. Not all platforms will support
+the creation of new app domains, for example, .NET Core will not, so the method
+`AppDomain.CreateDomain` while being available in .NET Standard might throw
+`PlatformNotSupportedException`.
+
+The primary reason we expose this type in .NET Standard is because the usage is
+fairly high and typically not associated with creating new app domains but for
+interacting with the current app domain, such as registering an unhandled
+exception handla or asking for the application's base directory.
+
+## Is `MarshalByRefObject` (remoting) part of .NET Standard?
+
+We don't plan to add remoting support to .NET Standard. However, in order to
+avoid potential breaking changes, we'll have the `MarshalByRefObject` type as
+many other types derive from it but we will not expose any remoting-specific
+members on it, such as `CreateObjRef`.
+
+## Is `System.Data` part of .NET Standard?
+
+.NET Standard will contain the abstractions (`DbProvider`,
+`DbProviderFactories`, `DbConnection`, `IDbConnection` etc) as well as the the
+general ADO.NET facilities (`DataSet`, `DataTable` etc) APIs.
+
+We don't plan on adding any specific providers to .NET Standard as their
+applicability varies (e.g. it's a highly unlikely scenario to use the SQL Server
+client from an iOS device, but it would make sense to use a provider that can
+store data on the device, such as SQLite). The expectation is that those sit on
+top of .NET Standard or remain platform-specific.
+
+## Why is JSON.NET not part of .NET Standard?
+
+Today, the best library for dealing with JSON is JSON.NET. But by adding it to
```

Comment:
"The best" is debatable as many people prefer other libraries like ServiceStack.Text, Jil, SimpleJson for various reasons and whether it's the best or not is irrelevant for the argument you're trying to make in this section.

"One of the best" or "One of the most popular" is better IMHO.